core/src/main/java/com/linecorp/armeria/server/VirtualHostAnnotatedServiceBindingBuilder.java

```diff
@@ -166,6 +168,17 @@ public VirtualHostAnnotatedServiceBindingBuilder useBlockingTaskExecutor(boolean
         return this;
     }
 
+    /**
+     * Sets the query parameter delimiter. It is disabled by default.
+     *
+     * <p>Note that this delimiter works only when the resolve target class type is collection and the number
+     * of values of the query parameter is one.</p>
+     */
+    public VirtualHostAnnotatedServiceBindingBuilder useQueryDelimiter(String delimiter) {
```

Comment:
Not related with this PR, but we could make it less error prone to add a builder method to `AnnotatedServiceBindingBuilder` a little bit: https://github.com/line/armeria/issues/4205